src/frameworks/fastify.js

```diff
@@ -0,0 +1,44 @@
+function validate(validateRequest) {
+    return (request, reply) => {
+        const requestOptions = _getParameters(request);
+        return validateRequest(requestOptions).then(function (errors) {
+            if (errors) {
```

Comment:
If validation promise resolves with no errors, why would we throw anything?
However, empty array is truthy, so we need to account for that as well

Reply:
I've double-checked and we return undefined and not empty array in case there are no errors. Also we have exact same check for koa. Would you prefer it to be more defensive and also check for length?